pkgs/top-level/all-packages.nix

```diff
@@ -11456,9 +11456,6 @@ with pkgs;
 
   sleuthkit = callPackage ../tools/system/sleuthkit {};
 
-  # Not updated upstream since 2018, doesn't support qt newer than 5.12
-  sleepyhead = libsForQt512.callPackage ../applications/misc/sleepyhead {};
```

Comment:
Given that sleepyhead was forked into oscar, please add an entry on `aliases.nix` with an informative throw message.